src/components/page/page.module.scss

```diff
@@ -53,4 +54,14 @@
   .grid-media {
     grid-area: grid-media;
   }
+
+  .grid-image {
+    grid-area: grid-image;
+
+    img {
+      width: 100%;
+      height: auto;
```

Comment:
don't need it.  Height is implicitly set to auto if width is explicitly set (as in this case).

Reply:
Check. quick fix.